core_aiml/word_sub.py

```diff
@@ -57,41 +57,22 @@ def __init__(self, defaults={}):
 
     def __call__(self, match):
         """Handler invoked for each regex match."""
-        return self[match.group(0)]
+        ret = self[match.group(0)]

```

Comment:
Well spotted :) Both cases were from debugging the code to find some issues. Fixed in the last comit